src/djangooidc/views.py

```diff
@@ -78,9 +82,13 @@ def login_callback(request):
         if user:
             login(request, user)
             logger.info("Successfully logged in user %s" % user)
+            # Double login bug?
```

Comment:
```suggestion
```
Remember to remove this before you merge

Reply:
We actually want to keep this in for when we do tackle this ticket. Maybe I'll add the ticket number.